services/QuillLMS/lib/tasks/convert_activity_session_timetracking_data.rake

```diff
@@ -0,0 +1,12 @@
+namespace :activity_session_timetracking_data do
+  desc 'updates data that had formerly been saved as json in hstore to be parsable in jsonb'
+  task :convert => :environment do
+    ActivitySession.where("activity_sessions.data->'time_tracking' IS NOT NULL").each do |as|
+      if as.data['time_tracking'].instance_of? String
```

Comment:
I would double-check the type of `time_tracking` when accessing `data['time_tracking']`; namely string vs symbol.  Metabase might be doing some additional conversion behind the scenes on that key:

![Screen Shot 2021-06-02 at 7 38 54 AM](https://user-images.githubusercontent.com/2057805/120473958-972f9500-c375-11eb-9cd8-c2f26570f50b.png)

![Screen Shot 2021-06-02 at 7 35 37 AM](https://user-images.githubusercontent.com/2057805/120474138-d4942280-c375-11eb-8cef-d4a5d246136b.png)

Reply:
Thanks for asking! I did run this locally, with real data from having played through a bunch of Comprehension sessions, and it did work so I feel confident about it.

Reply:
I think the difference is that when you copy and paste the rails output into your console, it's just a hash; in this script it's a `ActiveRecord::ConnectionAdapters::PostgreSQL::OID::Jsonb`, which has string keys.

Reply:
Interesting.  Thank you for resolving confusing.